spec/lib/flapjack/gateways/pagerduty_spec.rb

```diff
@@ -72,10 +72,15 @@
        with(:headers => {'Authorization'=>'Token token=token123'}).
        to_return(:status => 200, :body => response.to_json, :headers => {})
 
+    stub_request(:get, "https://flpjck.pagerduty.com/api/v1/incidents?" +
```

Comment:
Yes, there's two specs in there now that look for acknowledgements:
- "looks for acknowledgements via the PagerDuty API"
- "looks for acknowledgements via the PagerDuty API with basic auth"

The former tests token auth